src/commands/command-getkeysandflags.json

```diff
@@ -0,0 +1,18 @@
+{
+    "GETKEYSANDFLAGS": {
```

Comment:
Maybe instead of adding a new command, we should use RESP3 attributes? 
I suppose it can cause difficulties to clients, which wanna send this command on a connection serving the app, and rather not switch protocol version.

Reply:
i agree.

i think that having another subcommand is an overkill here

Reply:
i'm afraid to break existing client libraries that use GETKEYS (maybe with RESP3), but didn't bother to support attributes (since redis didn't use them so far, this would be the first usage)

Reply:
@oranagra I agree this has more downsides than upsides:

* RESP3 only
* Potential compatibility issues
* Unnecessary overhead

Reply:
I would prefer keeping the new subcommand.